src/response.js

```diff
@@ -8,61 +8,55 @@ export class DropboxResponse {
   }
 }
 
-export function parseResponse(res) {
+function throwIfError(res) {
```

Comment:
As noted below, I think this function is not necessary. It returns nothing and we can use a general catch block for error handling.